lib/plaid/institutions.ex

```diff
@@ -120,12 +120,18 @@ defmodule Plaid.Institutions do
 
   @doc """
   Gets an institution by id.
+
+  Parameters
+  ```
+  %{
+    institution_id: "ins_109512", options: %{ include_optional_metadata: true }
+  }
+  ```
   """
-  @spec get_by_id(String.t(), config | nil) ::
+  @spec get_by_id(params, config | nil) ::
           {:ok, Plaid.Institutions.Institution.t()} | {:error, Plaid.Error.t()}
-  def get_by_id(id, config \\ %{}) do
+  def get_by_id(params, config \\ %{}) do
```

Comment:
it's a bit weird that get_by_id that take params expecting a  institution_id inside it.
I think it's more intuitive if we keep the institution id as first argument.  And expect caller can pass `params` inside config keyword list? like `get_by_id("ins_109512", params:  options: %{ include_optional_metadata: true }`?

Reply:
config keyword list is passed in a function for this purpose: ```Gets the `client_id` and `secret` from config argument or the library configuration.``` so I'd rather pass in a separate argument called options. just to make it consistent with the other functions.